scale_build/image/manifest.py

```diff
@@ -64,12 +64,13 @@ def build_release_manifest(update_file, update_file_checksum):
         }, f)
 
 
-def get_image_version():
+def get_image_version(vendor=None):
     if not os.path.exists(RELEASE_MANIFEST):
         raise CallError(f'{RELEASE_MANIFEST!r} does not exist')
 
     with open(RELEASE_MANIFEST) as f:
-        return json.load(f)['version']
+        vendor = f"-{vendor}" if vendor else ""
```

Comment:
Can we please be consistent with quotes usage ? i.e if file is using single quotes, let's use single quotes and double quotes otherwise